### PR TITLE
Fix External Network APIs

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr fixes some missing points and bugs detected integrating various external network steps

# How Has This Been Tested?

- [x] locally

### CODE REVIEW

File: cmd/liqo-controller-manager/main.go [hunk 1]
```diff
@@ -217,6 +221,10 @@ func main() {
 	// External network parameters
 	flag.Var(&gatewayServerResources, "gateway-server-resources", "The list of resource types that implements the gateway server. They must be in the form <group>/<version>/<resource>")
 	flag.Var(&gatewayClientResources, "gateway-client-resources", "The list of resource types that implements the gateway client. They must be in the form <group>/<version>/<resource>")
+	flag.StringVar(&wgGatewayServerClusterRoleName, "wg-gateway-server-cluster-role-name", "cluster-admin", // TODO: define a proper default value
+		"The name of the cluster role used by the wireguard gateway servers")
+	flag.StringVar(&wgGatewayClientClusterRoleName, "wg-gateway-client-cluster-role-name", "cluster-admin", // TODO: define a proper default value
+		"The name of the cluster role used by the wireguard gateway clients")
```
Comment (cheina97) [inline cmd/liqo-controller-manager/main.go:227]: If you want you can insert "liqo-gateway" as default clusterrole

File: deployments/liqo/templates/liqo-controller-manager-rbac.yaml [hunk 2]
```diff
@@ -25,6 +25,22 @@ roleRef:
...
+  labels:
+    {{- include "liqo.labels" $ctrlManagerConfig | nindent 4 }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "liqo.prefixedName" $ctrlManagerConfig }}
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (cheina97) [inline deployments/liqo/templates/liqo-controller-manager-rbac.yaml:42]: You can insert liqo-gateway here

**Review by cheina97 — APPROVED** [review]

**Review by fra98 — APPROVED** [review]

**Pull request merged**